client/X11/xf_event.c

```diff
@@ -839,7 +839,7 @@ static BOOL xf_event_MapNotify(xfContext* xfc, const XMapEvent* event, BOOL app)
 			 * Doing this here would inhibit the ability to restore a maximized window
 			 * that is minimized back to the maximized state
 			 */
-			xf_rail_send_client_system_command(xfc, appWindow->windowId, SC_RESTORE);
+			// xf_rail_send_client_system_command(xfc, appWindow->windowId, SC_RESTORE);
```

Comment:
Please add an explanation for this change

Reply:
it is redundancy, all window state change (maximized, minimized, restore) should be handled in xf_event_PropertyNotify. xf_event_MapNotify should stay the same style with xf_event_UnmapNotify in app mode, just update is_mapped member.

a window is mapped does not mean it should be shown restore or other state, so i think (appWindow->rail_state == WINDOW_SHOW)  is also redundancy.